.editorconfig

```diff
@@ -4,11 +4,38 @@ end_of_line = "lf"
 
 [*.cs]
 indent_style = tab
+indent_size = 4
 csharp_new_line_before_open_brace = types,methods
 csharp_new_line_before_else = false
 csharp_new_line_before_catch = false
 csharp_new_line_before_finally = false
+csharp_new_line_before_members_in_object_initializers = true
+csharp_new_line_before_members_in_anonymous_types = true
+csharp_new_line_between_query_expression_clauses = true
+
 csharp_space_after_keywords_in_control_flow_statements = true
 csharp_space_between_method_declaration_name_and_open_parenthesis = true
+csharp_space_between_method_declaration_name_and_open_parenthesis = true
 csharp_space_between_method_call_name_and_opening_parenthesis = true
-csharp_space_before_open_square_brackets = true
\ No newline at end of file
+csharp_space_before_open_square_brackets = false
```

Comment:
Should this stay `true` to follow mono coding guidelines?

Reply:
I'm open to tweaking this but there are unfortunately no different settings for array declaration and indexes. The original Mono coding guideline has different spacing for each which makes it difficult to set.

Mono guidelines

```csharp
int[] a = new object [10];
```

I went with the minimal changes to the sources and the matching default VS settings.